docs/embark-skills.rst

```diff
@@ -3,7 +3,7 @@ embark-skills
 
 .. dfhack-tool::
     :summary: Adjust dwarves' skills when embarking.
-    :tags: unavailable
+    :tags: fort units
```

Comment:
```suggestion
    :tags: embark armok units
```

Reply:
this matches what we had before we marked this tool as `unavailable` (master list here: https://docs.google.com/spreadsheets/d/1hiDlo8M_bB_1jE-5HRs2RrrA_VZ4cRu9VXaTctX_nwk/edit#gid=1774645373)